src/extension.ts

```diff
@@ -5,19 +5,19 @@ import { ExtensionContext } from 'vscode'
 import { Client } from './components/Client'
 import { Connection, InfluxDBTreeDataProvider } from './components/connections/Connection'
 
-let client : Client
+let languageClient : Client
```

Comment:
I was a little confused here, before, but this change is much more readable without the diff.

Reply:
Heh. I, too, was confused. What is `Client`? Why isn't it used anywhere else? Turns out that when I removed it in a blind rage, I killed the lsp client. 😂 When I added it back in, I renamed it so I knew not to remove it in the future.